<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="kValueSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="minimum">
     <number>4</number>
    </property>
    <property name="maximum">
     <number>16</number>
    </property>
    <property name="sliderPosition">
     <number>6</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="applyButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kanonText">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>k-anonimizer</string>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>0</y>
      <width>16</width>
      <height>561</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="kValueHolder">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="imageHolder">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="statsHolder">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>171</width>
      <height>391</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Stats:
No. of Nodes :
No. of Edges  :
avg. k	    :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_Node"/>
    <addaction name="actionAdd_Edge"/>
    <addaction name="separator"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionRemove_Files"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocs"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Node">
   <property name="text">
    <string>Add Node</string>
   </property>
  </action>
  <action name="actionAdd_Edge">
   <property name="text">
    <string>Add Edge</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDocs">
   <property name="text">
    <string>Docs</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRemove_Files">
   <property name="text">
    <string>Remove Files</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>kValueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>kValueHolder</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
